spec/lib/sm/api/folders_spec.rb

```diff
@@ -1,75 +1,58 @@
 # frozen_string_literal: true
 require 'rails_helper'
 require 'sm/client'
-require 'support/sm_client_helpers'
 
-describe SM::Client do
-  include SM::ClientHelpers
-
-  subject(:client) { authenticated_client }
-
-  describe 'get_folders' do
-    context 'with valid session and configuration' do
-      it 'gets a collection of folders' do
-        VCR.use_cassette('sm/folders/10616687/index') do
-          client_response = client.get_folders
-          expect(client_response).to be_a(Common::Collection)
-          expect(client_response.type).to eq(Folder)
+describe 'sm client' do
+  describe 'folders' do
+    before(:all) do
+      VCR.use_cassette 'sm_client/session', record: :new_episodes do
+        @client ||= begin
+          client = SM::Client.new(session: { user_id: ENV['MHV_SM_USER_ID'] })
+          client.authenticate
```

Comment:
It does, good catch. This won't hurt anything though, I'll clean it up in next PR.